frontend/packages/form/README.md

```diff
@@ -1,8 +1,12 @@
-# Vaadin Common Frontend
+# Vaadin Form
+
+[API documentation ↗](https://vaadin.com/docs/latest/fusion/forms/appendix-client-side-form-binding-reference)
 
 [![Latest Stable Version](https://img.shields.io/npm/v/@vaadin/form.svg)](https://www.npmjs.com/package/@vaadin/form)
 
-A tool to connect HTML forms with the Fusion backend seamlessly.
+A set of utilities for building forms in TypeScript, including data binding, validation, code generation, backend integration, Lit directives, and more.
```

Comment:
Lit is actually an essential part of @vaadin/form, maybe better to say
A set of utilities for building forms with TypeScript and [Lit](https://lit.dev/), including data binding, validation, code generation, backend integration, Lit directives, and more.